package.json

```diff
@@ -122,7 +122,10 @@
     "nypr-django-for-ember": ">=0.2.0",
     "nypr-metrics": "<0.1.0",
     "nypr-publisher-lib": ">=0.3.5",
-    "nypr-ui": "nypublicradio/nypr-ui#kate/RT-941",
+    "nypr-ui": "^0.2.19",
+    "customizr": "Modernizr/customizr#e5890ed4b4fedb69da50ffbae39577788970d8ee"
```

Comment:
I'm not actually sure how customizr got added to resolutions (did yarn do it automagically?), but I assume it's fine to be there.